frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -248,11 +252,13 @@ export const CreateSubscriptionYAML: React.FC<CreateSubscriptionYAMLProps> = (pr
 };
 
 const tableColumnClasses = [
-  classNames('col-lg-3', 'col-md-3', 'col-sm-4', 'col-xs-6'),
-  classNames('col-lg-2', 'col-md-3', 'col-sm-4', 'col-xs-6'),
-  classNames('col-lg-2', 'col-md-3', 'col-sm-4', 'hidden-xs'),
-  classNames('col-lg-3', 'col-md-3', 'hidden-sm', 'hidden-xs'),
-  classNames('col-lg-2', 'hidden-md', 'hidden-sm', 'hidden-xs'),
+  classNames('pf-m-visible-on-xs'),
```

Comment:
I think you could just leave the `pf-m-visible-on-xs` classname off.

```suggestion
  '',
```

Reply:
yep, this works